src/status_im/ui/components/numpad/views.cljs

```diff
@@ -0,0 +1,34 @@
+(ns status-im.ui.components.numpad.views
+  (:require [status-im.ui.components.numpad.styles :as styles]
+            [status-im.ui.components.icons.vector-icons :as icons]
+            [status-im.ui.components.colors :as colors]
+            [status-im.ui.components.react :as react]))
+
+(defn number-view
```

Comment:
is it the same namepad again?

Reply:
Yes, but this will be now in a separate PR, together with PINs logic. Removed from this PR now.